src/Pointer.js

```diff
@@ -72,6 +72,23 @@ export class Pointer {
       this._nextTick = trigger();
     };
 
+    const _mouseLeave = (e) => {
+      if (e.target === this.config.root) {
+      Object.assign(this.previousProgress, this.currentProgress || this.progress);
+      this.progress.active = 0;
```

Comment:
Why not simply:
```suggestion
      this.progress.active = true;
```
?

Reply:
That's exactly what I initially did and saw that somewhere along the flow it was changed to a number. I thought that maybe it's a wanted behavior of mouse progress to use only numbers.

Reply:
OK, I think it's happening here: https://github.com/wix-incubator/kuliso/blob/master/src/Pointer.js#L138
But then it's not only changed into a number, it's also interpolated.
Let's do a special case there: `if (key === 'active') acc[key] = value`, for now